.ci/pytorch/multigpu-test.sh

```diff
@@ -50,6 +50,9 @@ time python test/run_test.py --verbose -i distributed/tensor/parallel/test_tp_ra
 # FSDP2 tests
 time python test/run_test.py --verbose -i distributed/_composable/fsdp/test_fully_shard_training -- -k test_2d_mlp_with_nd_mesh
 
+# Pipelining composability tests
+time python test/run_test.py --verbose -i distributed/pipelining/test_composability.py
```

Comment:
Is this the only pipelining test requiring 4+ GPUs right now?

Reply:
yes, I think so